examples/transactions.c

```diff
@@ -36,8 +36,9 @@
  * The transaction is committed every 5 seconds or 100 messages, whichever
  * comes first. As the transaction is committed a new transaction is started.
  *
- * @remark This example does not yet support incremental rebalancing and thus
- *         not the cooperative-sticky partition.assignment.strategy.
+ * This example makes use of incremental rebalancing (KIP-429) and the
+ * cooperative-sticky partition.assignment.strategy on the consumer, providing
+ * hitless rebalances.
```

Comment:
👊 🤜

Reply:
hitless doesn't sound right.

maybe "optimized, non stop-the-world rebalances."? (not great either, but i think a step forward)

Reply:
Meh.